kube-runtime/src/reflector/store.rs

```diff
@@ -60,6 +62,7 @@ where
         let (ready_tx, ready_rx) = DelayedInit::new();
         Writer {
             store: Default::default(),
+            buffer: Default::default(),
```

Comment:
does the buffer also need to be RwLocked? it is only read from internally

Reply:
I could remove the `RwLock`, see: https://github.com/kube-rs/kube/pull/1494/commits/c6100e1fe0ed1a0efa51c2aeba087af7ae9ff73e

Reply:
if the borrow checker is happy then that looks good